packages/api/utils/emptyPortfolioMiddleware.ts

```diff
@@ -0,0 +1,40 @@
+import middy from "@middy/core";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { CloudServiceProvider } from "../models/CloudServiceProvider";
+
+function isRequestBodyEmptyString(body: string | null): boolean {
+  return body?.length === 0;
+}
```

Comment:
Do we want to cover an empty JSON body as well (e.g., `"{}"`)? In previous routes, an empty request body would be sent such as`{}`, so I was wondering if we wanted to follow that pattern here or simply send an empty string.

Reply:
The API spec lists two individual content/content types for request bodies, one for JSON, and one for plain text.  An empty JSON body will get caught by the validator and return 400.

I've included examples of the empty string request and a full portfolio JSON request in the ticket.

```
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/Portfolio'
          text/plain:
            schema:
              type: string
              example: ''
```